docs/resources/oms_migration_sync_task.md

```diff
@@ -0,0 +1,180 @@
+  in the object list. Value options:
+
+  + **size_last_modified**: the system checks object consistency with object size and last modification time.
+  If a source object is as large as but was last modified earlier than its paired destination object, the system
+  considers the source object does not need to be migrated or has been already migrated successfully.
+
+  + **crc64**: this option is only available for migration on Huawei Cloud or from Alibaba Cloud or Tencent Cloud. If
+  a source object and its paired destination object have CRC64 checksums, the checksums are checked. Otherwise, their
+  sizes and last modification times are checked.
+
+  + **no_check**: this option is only available for migration of HTTP/HTTPS data. This option takes effect for source
+  objects whose sizes cannot be obtained using the content-length field in the standard HTTP protocol. These source
+  objects will overwrite their paired destination objects directly.
+  If the size of a source object can be obtained, its size and last modification time will be checked.
+
+  The default value is **size_last_modified**. Changing this creates a new resource.
+
+* `description` - (Optional, String, ForceNew) Specifies the description of the synchronization task.
+  Changing this creates a new resource.
+
+* `enable_kms` - (Optional, Bool, ForceNew) Specifies whether to enable the KMS encryption function.
+  Default value: **false**. Changing this creates a new resource.
+
+* `enable_metadata_migration` - (Optional, Bool, ForceNew) Specifies whether metadata migration is enabled.
+  Default value: **false**. Even if disabled, the ContentType metadata will still be migrated
+  to ensure a successful migration. Changing this creates a new resource.
+
+* `enable_restore` - (Optional, Bool, ForceNew) Specifies whether to automatically restore the archive data. If enabled,
+  archive data is automatically restored and migrated. Default value: **false**. Changing this creates a new resource.
+
+* `source_cdn` - (Optional, List, ForceNew) Specifies the CDN information. If this parameter is contained,
+  using CDN to download source data is supported, the source objects to be migrated are obtained from the CDN domain
+  name during migration. Changing this creates a new resource.
+  The [source_cdn](#block--source_cdn) structure is documented below.
+
+* `src_cloud_type` - (Optional, String, ForceNew) Specifies the source cloud service provider. Value options:
+  **AWS**, **Azure**, **Aliyun**, **Tencent**, **HuaweiCloud**, **QingCloud**, **KingsoftCloud**, **Baidu**,
+  **Qiniu**, **URLSource** and **UCloud**. Default value: **Aliyun**. Changing this creates a new resource.
+
+* `action` - (Optional, String) Specifies the action for migration synchronization task. Value options:
+  + **start**: Resume a paused migration synchronization task.
+  + **stop**:  Pause a migration synchronization task.
```

Comment:
without the description of the default value
```
Default value: **start**.
```

Reply:
done